lib/index.css

```diff
@@ -0,0 +1,43 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) Jupyter Development Team.
+| Distributed under the terms of the Modified BSD License.
+|----------------------------------------------------------------------------*/
+.jp-FileBrowser-Item {
+  color: #2F2F2F;
+  display: inline-block;
+  font: 14px Helvetica, Arial, sans-serif;
+}
+
+.jp-FileBrowser-ListItem > div {
```

Comment:
That appears to be the convention in other Phosphor repos, lowercase unless it is the name of a `class`, and use hyphen separators.